### DISCUSSION TITLE
Is there a way to generate the final sound file faster?

### DISCUSSION THREAD

**Discussion by gexagonr** [neonbjb/tortoise-tts#213] [General]
Im using a RTX 3080, and when I want output a paragraph with the 'read' command it takes about 2 hours on 'fast' setting. Takes 15-20 minutes on 'ultra_fast'. Is there some setting/configuration I am missing or not doing?

**Comment by neonbjb** [comment]
You are probably not using the GPU. Google around for some "pytorch GPU set-up" tutorials to make sure you are.

**Reply by gexagonr** [reply]
That was it thanks!

**Reply by Lonas13** [reply]
Can you elaborate on how you fixed it?

**Comment by lavnishhh** [comment]
Hi, you seem to have successfully switched it to GPU. Can you help me with it too? I have a RTX3050 and it takes about 4-5 minutes just for 7-9 seconds of audio generation. How long does it take you?

**Reply by lavnishhh** [reply]
It took me about 45 minutes to generate a clip of length 91 seconds in parts. Is this normal?
